network/src/commonMain/kotlin/com/wire/kalium/network/utils/NetworkResponse.kt

```diff
@@ -18,6 +18,7 @@ sealed class NetworkResponse<out T : Any> {
             // small issue here where keys are converted to small case letters
             // this is an issue for ktor to solve
             httpResponse.headers.toMap()
+                .mapKeys { headerEntry -> headerEntry.key.lowercase() }
```

Comment:
Comments around this code suggests we should stop exposing `headers` and `cookies` on `NetworkResponse` so maybe we should use KTORs header/cookie API directly which is case insensitive.